packages/block-editor/src/components/list-view/use-block-selection.js

```diff
@@ -32,7 +32,6 @@ export default function useBlockSelection() {
 	const updateBlockSelection = useCallback(
 		async ( event, clientId, destinationClientId ) => {
 			if ( ! event?.shiftKey ) {
-				await clearSelectedBlock();
```

Comment:
Why was this change necessary? Tested selection for single and multiple blocks and seems to work fine. Did you run into any issue?

Reply:
Hi @oandregal, without this change, if we enable modify as blocks when we select an inner block, the modify as blocks stop. That happens because modifying as blocks stops when the block is unselected, and with this line, there is no block selected for some time.